components/brave_new_tab_ui/components/app.tsx

```diff
@@ -12,6 +13,7 @@ import NewTabPage from './newTab'
 
 // Utils
 import * as newTabActions from '../actions/new_tab_actions'
+import * as PreferencesAPI from '../api/preferences'
```

Comment:
can we not capitalize this call? s/PreferencesAPI/preferencesAPI

Reply:
I don't think we've defined the standard across our codebase have we @cezaraugusto ? I would suggest we follow the practice of:
- modules that export a single thing get `camelCase`
- modules that export a library of function get `PascalCase`

Then you get `PascalCaseAPI.camelCaseFunction(blah)`, and we know what is a _thing_ and what is a _library_ of things.

But if we have defined a different consistent pattern, then I'm happy to follow.

Reply:
I disagree but commit with your thought. current standard is an unspoken rule of always camelcase but I guess the practice you're mentioning comes from our c++ codebase? if this rule helps you and maybe others to identify faster what's going on, I'm game. 

also would be great if we could hack an eslint file or something similiar to spot wrong cases

Reply:
The practice comes not from c++ but from my other JS work, but if no-one's been doing that and doesn't see the value in differentiating the different scenarios then ok

Reply:
A brief search on what others are doing yielded this which looks informed by the same thinking - modules and classes which have members get PascalCase, and members or single functions get camelCase: https://stackoverflow.com/questions/29601960/typescript-use-pascalcasing-or-camelcasing-for-module-names

Reply:
if you see value on this there's no reason not to do it :) could you please add this rule to a wiki page so others could see and follow? also worth sharing in front-end channel. I'm resolving this one and +1ning